thirdparty_builtin/CMakeLists.txt

```diff
@@ -86,17 +86,33 @@ if(ENABLE_TESTS)
     #
     
     if(ENABLE_GTEST)
+        
+        set(gtest_extra_flags "")
+        
+        if(COMPILER_FAMILY_IS_MSVC)
```

Comment:
You can use the ``blt_append_custom_compiler_flag`` macro.

E.g. 
```cmake
blt_append_custom_compiler_flag( 
    FLAGS_VAR gtest_extra_flags
    DEFAULT " " 
    MSVC  "/wd4251 /wd4251"
    )
```

Reply:
except, we only want it to be associated with gtest + gmock -- so we need to have it as part of the info passed to btl register library

Reply:
The ``blt_append_custom_compiler_flag`` macro only appends the given flags into the provided variable, and does so in a way that hides the compiler specific cmake logic.  You can then use that variable in the same way, by passing it to the ``blt_register_library`` for gmock and gtest. There are no other side effects.

This has the added benefit of centralizing this logic, so that fixes can be more easily incorporated throughout our build system.  E.g. see issue #81, where we need to add support for the intel compiler through MSVC.

Reply:
Sorry, I misunderstood - I will definitely move to use that.